tensorflow/core/util/mkl_util.h

```diff
@@ -1886,9 +1972,18 @@ class MklPrimitiveFactory {
  private:
   static inline LRUCache<MklPrimitive>& GetLRUCache() {
     static const int kCapacity = 1024;  // cache capacity
+#ifndef DNNL_AARCH64_USE_ACL
     static thread_local LRUCache<MklPrimitive> lru_cache_(kCapacity);
+#else
+    static LRUCache<MklPrimitive> lru_cache_(kCapacity);
+#endif
     return lru_cache_;
   }
+
+#ifdef DNNL_AARCH64_USE_ACL
+  mutex mtx_;
```

Comment:
Nit: Let's make the naming clear what we are using it for. Let's also use `mu` to abbreviate for mutex to be consistent with other files.
```suggestion
  mutex primitive_creation_mu_;
```